Add icon 1

Aichnerc opened the pull request:
**Checklist:**

 <!--- Make sure your PR is documented and tested before submission. Put an `x` in all the boxes that apply: -->
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?


**Type of Changes:**

 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)


**What is the current behavior? (link to any open issues here)**
- Currently, the default React icon is used for our favicon. #6 


**What is the new behavior (if this is a feature change)?**
- A new icon has been designed and implemented. #6

Aichnerc committed:
Add new standards

The new guidelines for commenting style and
folder structure were applied.

Aichnerc committed:
Add Outrun Icon

The new React Outrun icon has been added
to this project.

kleberbaum commented on public/fav/browserconfig.xml:
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig><msapplication><tile><square70x70logo src="/ms-icon-70x70.png"/><square150x150logo src="/ms-icon-150x150.png"/><square310x310logo src="/ms-icon-310x310.png"/><TileColor>#ffffff</TileColor></tile></msapplication></browserconfig>
```
It would be much better with some line breaks.

Aichnerc replied:
The formatting of this document has been changed with my latest commit.

kleberbaum reviewed (changes requested):
After this it should be ok to merge.

Aichnerc committed:
Format document

The browserconfig.xml for our icons has been
formatted so it fits our coding guidelines.

Aichnerc reviewed (commented):
Everything seems clean now.

kleberbaum reviewed (approved):
Should be ok to merge.
